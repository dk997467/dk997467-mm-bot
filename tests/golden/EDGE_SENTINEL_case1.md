EDGE SENTINEL REPORT

| symbol | gross_bps | fees_eff_bps | adverse_bps | slippage_bps | inventory_bps | net_bps | fills | turnover_usd |
|--------|-----------|--------------|-------------|---------------|----------------|---------|-------|---------------|
| BTCUSDT | 0.300000 | 0.300000 | 0.500000 | 0.700000 | 0.000000 | -0.600000 | 2 | 201.000000 |
| ETHUSDT | 0.500000 | 0.300000 | 0.300000 | 0.700000 | 0.000000 | -0.400000 | 2 | 802.000000 |

Advice:
- tune throttle: raise min_interval_ms
- lower micro impact_cap_ratio


