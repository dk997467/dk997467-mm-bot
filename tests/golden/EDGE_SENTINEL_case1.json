{"advice":["tune throttle: raise min_interval_ms","increase vip tilt cap mildly"],"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"summary":{"buckets":[{"adverse_bps":0.25,"bucket":"1970-01-01T00:00Z","fees_eff_bps":0.15000000000000002,"fills":2.0,"gross_bps":0.15000000000000002,"inventory_bps":0.0,"net_bps":-0.6,"slippage_bps":0.35,"symbol":"BTCUSDT","turnover_usd":201.0},{"adverse_bps":0.15000000000000002,"bucket":"1970-01-01T00:00Z","fees_eff_bps":0.15000000000000002,"fills":2.0,"gross_bps":0.25,"inventory_bps":0.0,"net_bps":-0.4,"slippage_bps":0.35,"symbol":"ETHUSDT","turnover_usd":802.0}],"symbols":{"BTCUSDT":{"adverse_bps":1.3,"fees_eff_bps":0.7,"fills":5.0,"gross_bps":0.8,"inventory_bps":0.0,"net_bps":-0.6,"slippage_bps":1.8,"turnover_usd":502.0},"ETHUSDT":{"adverse_bps":0.7,"fees_eff_bps":0.7,"fills":5.0,"gross_bps":1.3,"inventory_bps":0.0,"net_bps":-0.33999999999999997,"slippage_bps":1.6,"turnover_usd":2004.0}}},"top":{"contributors_by_component":{"adverse_bps":["BTCUSDT","ETHUSDT"],"fees_eff_bps":["BTCUSDT","ETHUSDT"],"inventory_bps":["BTCUSDT","ETHUSDT"],"slippage_bps":["BTCUSDT","ETHUSDT"]},"top_buckets_by_net_drop":["1970-01-01T00:00Z"],"top_symbols_by_net_drop":["BTCUSDT","ETHUSDT"]}}
